Document that GitHub username is added to internal.logins (#22782)
gravitational/teleport: docs/pages/access-controls/sso/github-sso.mdx

  able to determine team memberships for these organizations.
</Admonition>

> After authentication Teleport will add the GitHub username to that user's

Reviewer: ```suggestion
After a user authenticates, Teleport will add the user's GitHub username to their
```

> After authentication Teleport will create a temporary GitHub type user within Teleport for the duration of the 

Reviewer 2: I don't love this sentence.

- "Teleport will create a user within Teleport" is a little confusing
- "GitHub type user" is not a term I'd expect the reader to understand

Author: Removed and simplified language.

> `internal.logins` trait for their session. The preset `access` role has this

Reviewer: ```suggestion
`internal.logins` trait for their Teleport session. The preset `access` role has this
```

> session. The GitHub username is automatically added to that user's `internal.logins` variable

Reviewer 2: Technically that's a _trait_, not a variable, right?

Author: The `internal.logins` is the trait to the user.  Within here we refer to `internal.logins` as a variable for its usage in roles.  I've updated to describe as a trait variable that I believe is more clear.

> After authentication Teleport will add the GitHub username
> to that user's `internal.logins` trait for their session. The preset `access`
> role has this trait variable configured to include the GitHub user as an
> authorized SSH login. Here is an example role configuration snippet using the trait variable. 

Reviewer 3: Adjusted line breaks :

```suggestion
After authentication Teleport will add the GitHub username to that user's
`internal.logins` trait for their session. The preset `access` role has this
trait variable configured to include the GitHub user as an authorized SSH login.
Here is an example role configuration snippet using the trait variable. 
```

> Here is an example role configuration snippet using the trait variable. 

Reviewer: ```suggestion
Here is an example role configuration snippet using the trait variable:
```

> usable within roles. The default `access` role has this variable that will automatically include the

Reviewer 2: ```suggestion
usable within roles. The preset `access` role has this variable that will automatically include the
```

> usable within roles. The default `access` role has this variable that would automatically include the

Reviewer 3: would implies "used to".

```suggestion
usable within roles. The default `access` role has this variable that will automatically include the
```

> GitHub user as an authorized SSH login. Here is an example role configuration snippet using the variable. 

Reviewer 2: ```suggestion
GitHub user as an authorized SSH login. Here is an example role configuration snippet using the trait. 
```
